testing/test_hookrelay.py

```diff
@@ -87,9 +87,17 @@ class Plugin3(object):
         def hello(self, arg):
             return None
 
+    class Plugin4(object):
```

Comment:
Perhaps we should add a new test which also changes the `outcome` result to make sure?

I plan to test this branch in pytest later today as well.

Reply:
@nicoddemus yes good call.
I'll add a *force result* test.